index.bs

```diff
@@ -13324,10 +13369,8 @@ Each {{DOMException}} object has an associated <dfn for="DOMException">name</dfn
 The <dfn constructor for="DOMException"><code>DOMException(|message|, |name|)</code></dfn>
 constructor, when invoked, must run these steps:
 
-1. Let |e| be a new {{DOMException}} object.
-1. Set |e|'s [=DOMException/name=] to |name|.
-1. Set |e|'s [=DOMException/message=] to |message|.
-1. Return |e|.
+1. Set <emu-val>this</emu-val>'s [=DOMException/name=] to |name|.
```

Comment:
Technically "this" is not really defined that well; that's https://www.w3.org/Bugs/Public/show_bug.cgi?id=27301. However I support landing this as-is and as a follow-up working on that additional rigorization project. The text you have that says "as the this value" is pretty good for now.

In particular I think we'd want to define "this value" or "this" as something linkable, for both methods and constructors, and encourage a typography that either leaves it un-marked-up or cross-links to the Web IDL definition.

Reply:
I'd prefer "this" over "this value" as it's extremely common.